lib/js/components/SelectList/SelectListItem/SelectListItem.vue

```diff
@@ -0,0 +1,148 @@
+<template>
+	<div
+		v-ripple
+		class="selectListItem"
+		:class="{ '-loading': isLoading, '-selected': isSelected, [`-${size}`]: true }"
+	>
+		<ds-icon
+			v-if="iconLeft || isLoading"
+			class="selectListItem__iconLeft"
+			:icon="isLoading ? ICONS.FAD_SPINNER_THIRD : iconLeft"
+			:size="ICON_SIZES.X_SMALL"
+			:spinning="isLoading"
+		/>
+		<span class="selectListItem__text">{{ label }}</span>
```

Comment:
should we add `title` attribute to cover too long labels dispayed with ellipsis?